database/migrations/2023_05_02_170519_create_formas_pagamento_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('formas_pagamento', function (Blueprint $table) {
             $table->id();
-            $table->string('tipo')->nullable(false);
+            $table->string('tipo');
```

Comment:
Como tu tá usando string para a coluna tipo, ao invés do enum ( string é preferível mesmo ), deveria ser validado, no ```app/Http/Requests/StoreBancaRequest.php```, quais dados vão poder ser enviados para não gerar erros e inconsistência. No caso, os mesmo do ```database/seeders/FormaPagamentoSeeder.php```.

Reply:
Na verdade o $request->forma_pagamento que é recebido no store de banca é uma string com o id de todas as formas de pagamento que a banca terá separados por vírgula. Dessa forma acho que não teria como usar um validate tipo "in" como usamos para validar o tipo_unidade de venda. A solução seria colocar esse trecho do código em um try-catch e fazer o tratamento da exceção?

Reply:
No seeder apenas é salvo os diferentes tipos de pagamento, que serão fixos. Em seguida uso o id para associar as bancas aos tipos de pagamento, o mesmo vale para as bancas. Acho melhor pegar pelo id mesmo para evitar erros caso o nome de alguma forma de pagamento seja modificado.